.github/workflows/ci.yml

```diff
@@ -70,3 +72,31 @@ jobs:
       - run: mix credo --strict --format=oneline
 
       - run: mix test --warnings-as-errors --cover
+
+      - name: Restore PLT cache
+        id: plt_cache_restore
+        uses: actions/cache/restore@v3
+        with:
+          key: |
+            plts-${{ runner.os }}-${{ steps.beam.outputs.otp-version }}-${{ steps.beam.outputs.elixir-version }}-${{ hashFiles('**/mix.lock') }}
+          restore-keys: |
+            plts-${{ runner.os }}-${{ steps.beam.outputs.otp-version }}-${{ steps.beam.outputs.elixir-version }}-
+          path: |
+            priv/plts
+
+      - name: Create PLTs
+        if: steps.plt_cache_restore.outputs.cache-hit != 'true'
+        run: mix dialyzer --plt
+
+      - name: Save PLT cache
+        id: plt_cache_save
+        if: steps.plt_cache_restore.outputs.cache-hit != 'true'
+        uses: actions/cache/save@v3
+        with:
+          key: |
+            plts-${{ runner.os }}-${{ steps.beam.outputs.otp-version }}-${{ steps.beam.outputs.elixir-version }}-${{ hashFiles('**/mix.lock') }}
+          path: |
+            priv/plts
+
+      - name: Run dialyzer
+        run: mix dialyzer --format github --format dialyxir
```

Comment:
Duplicate flags:

```suggestion
        run: mix dialyzer --format github
```

Reply:
This is the suggested approach from dialyzir

Their documentation:

```
 - name: Run dialyzer
    # Two formats are included for ease of debugging and it is lightly recommended to use both, see https://github.com/jeremyjh/dialyxir/issues/530 for reasoning
    # --format github is helpful to print the warnings in a way that GitHub understands and can place on the /files page of a PR
    # --format dialyxir allows the raw GitHub actions logs to be useful because they have the full warning printed
    run: mix dialyzer --format github --format dialyxir
```

https://github.com/jeremyjh/dialyxir/blob/master/docs/github_actions.md